package.json

```diff
@@ -15,7 +15,7 @@
     "axios": "^1.7.7",
     "axios-mock-adapter": "^1.22.0",
     "concurrently": "^8.2.2",
-    "eslint": "^9.2.0",
+    "eslint": "^8.57.0",
```

Comment:
~~Why did eslint get downgraded?~~ See that it had to do with peer dependency. You can fix this by instead upgrading typescript-eslint to the latest version.

Reply:
Upgraded typescript-eslint to the latest version